decidim-admin/app/views/layouts/decidim/admin/participatory_process.html.erb

```diff
@@ -1,21 +1,21 @@
 <% content_for :secondary_nav do %>
   <div class="secondary-nav secondary-nav--subnav">
     <ul>
-      <% if can? :update, participatory_process %>
-        <li <% if is_active_link?(decidim_admin.edit_participatory_process_path(participatory_process)) %> class="is-active" <% end %>>
-          <%= aria_selected_link_to t("info", scope: "decidim.admin.menu.participatory_processes_submenu"), decidim_admin.edit_participatory_process_path(participatory_process) %>
+      <% if can? :update, current_participatory_process %>
```

Comment:
All this could probably be refactored into [`Decidim::Menu`](https://github.com/decidim/decidim/blob/master/decidim-core/lib/decidim/menu.rb) at some point, right?